src/TgSharp.Core/Network/MtProtoSender.cs

```diff
@@ -547,10 +538,50 @@ private bool HandleContainer(ulong messageId, int sequence, BinaryReader message
 
             return false;
         }
+        #endregion
+
+        #region Helper functions
```

Comment:
regions are a smell, create a new class

Reply:
I'll just remove the regions just wanted to be able to collapse multiple functions at once while I was working on them